### DISCUSSION TITLE
Commerical use

### DISCUSSION THREAD

**Discussion by madfortech** [saadeghi/daisyui#2196] [Q&A] [answered]
I use this package Laravel project. and sell. Ex: theme forest
my question is. 
can i have rights? for commercial use?

**Comment by saadeghi** [comment, marked as answer]
Yes. It's MIT license.  
https://choosealicense.com/licenses/mit/

You can use it commercially for free if you include a notice for license and copyright (which will happen by default if you're using daisyUI as a Node dependency. No need to do anything)

Let me know if you have any questions
